mounts/org.osbuild.ext4

```diff
@@ -30,7 +30,9 @@ SCHEMA_2 = """
     "properties": {
       "readonly": {
         "description": "mount the source as a readonly device",
-        "type": "boolean"
+        "type": "boolean",
+        "default": false
+
```

Comment:
extra newline

Reply:
I'm glad you have my back!

Reply:
I'm sorry for introducing them :sweat_smile: